models/user.js

```diff
@@ -4,11 +4,20 @@ var	passportLocal=require("passport-local-mongoose");
 var userSchema=new mongoose.Schema({
 
 	//TODO: DEFINE USERNAME AND PASSSWORD ATTRIBUTES
-
-
-    loaned_books:[
-        //TODO: embed reference to id's of book copies loaned by this particular user in this array
-    ]
+    // username: {
+    //     type: String,
+    //     required: true
+    // },
+    // password: {
+    //     type: String,
+    //     required: true
+    // },
```

Comment:
If you are accepting mail from user, make a field for it too

Reply:
Sure! I think i forgot that.